shell/packages/sandstorm-db/db.js

```diff
@@ -2583,15 +2574,15 @@ if (Meteor.isServer) {
           proof.status = "unverified";
         });
 
-        KeybaseProfiles.update(keyFingerprint, { $set: record }, { upsert: true });
+        this.collections.keybaseProfiles.update(keyFingerprint, { $set: record }, { upsert: true });
```

Comment:
`this` does not refer to the SandstormDb here because we're in a function(). Luckily I coincidentally noticed because an app update landed on my machine while I had run-dev going in a visible console window.

I think we need to be more cautious about these mass refactorings. We are very frequently breaking things and introducing bugs. We really need to develop much better tests before we can do this kind of stuff.